python/cuml/cluster/dbscan.pyx

```diff
@@ -188,6 +197,8 @@ class DBSCAN(Base):
         self.labels_ = cudf.Series(np.zeros(self.n_rows, dtype=np.int32))
         self.labels_array = self.labels_._column._data.to_gpu_array()
         cdef uintptr_t labels_ptr = self._get_ctype_ptr(self.labels_array)
+
```

Comment:
Stray spaces?

Reply:
I tend to write code in paragraph form and insert newlines when something appears too jumbled together to be as obvious as it could be. Perhaps my IDE inserted a space here.

Reply:
I tend to write code in paragraph form and insert newlines when something appears too jumbled together to be as obvious as it could be. Perhaps my IDE inserted the space here?